### PR TITLE
Fixed lint:frontend npm script

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] DX

## Description

The `lint:frontend` npm script was running but was resulting in a false positive for a successful lint of the front end. When I put this script in place, I didn't realize the folders needed to be in quotes. Oddly enough, it ran fine from the command line. Anyway, fixed.

## QA Instructions, Screenshots, Recordings

N/A

### UI accessibility concerns?

N/A

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: This is fixing a listing script only.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

A person banging an air ventilation system to get it working

### CODE REVIEW

File: app/javascript/.eslintrc.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ module.exports = {
   },
   plugins: ['import', 'react', 'jsx-a11y'],
   rules: {
+    'no-var': 'error',
```
Comment (nickytonline) [inline app/javascript/.eslintrc.js:46]: I added this override to the rule as the default for it was to warn only.

File: app/javascript/articles/components/CommentsList.jsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export const CommentsList = ({ comments = [], articlePath, totalCount }) => {
     return (
       <div className="crayons-story__comments">
         {comments.slice(0, numberOfCommentsToShow).map((comment) => {
-          return <CommentListItem comment={comment} />;
+          return <CommentListItem key={comment.id} comment={comment} />;
```
Comment (nickytonline) [inline app/javascript/articles/components/CommentsList.jsx:39]: Here and below are lint errors related to React requiring a `key` prop.

File: app/javascript/utilities/color/WCAGColorContrast.js [hunk 3]
```diff
@@ -12,13 +12,16 @@ export const WCAGColorContrast = {
    * @return {Number}
    */
   ratio(rgb1, rgb2) {
+    let sRGB1;
```
Comment (nickytonline) [inline app/javascript/utilities/color/WCAGColorContrast.js:15]: Fixed `var` being used, which kickstarted this whole PR. 😎

**Review by Ridhwana — APPROVED** [review]
Thanks @nickytonline this looks good to me 🚀

**Review by aitchiss — APPROVED** [review]
Nice!

**Pull request merged**